packages/pharos/src/components/link/pharos-link.ts

```diff
@@ -77,6 +77,13 @@ export class PharosLink extends FocusMixin(AnchorElement) {
   @property({ type: Boolean, reflect: true, attribute: 'no-hover' })
   public noHover = false;
 
+  /**
+   * Indicates if the link should render the visited link styling.
+   * @attr bold
```

Comment:
Be sure to update this.

Reply:
Updated in [ab9058a](https://github.com/ithaka/pharos/pull/409/commits/ab9058a239f6a6b1f2ce37a40e461ca48679f1a9)